build-shared/src/main/java/com/superzanti/serversync/server/ServerSetup.java

```diff
@@ -51,10 +51,10 @@ public ServerSetup() {
 
             Logger.log(String.format("Manifest files: %s", PrettyCollection.get(manifest.files)));
 
-            manifest.directories.stream().map(d -> ServerSync.rootDir.resolve(Paths.get(d.path))).forEach(p -> {
+            manifest.directories.stream().map(d -> ServerSyncUtility.rootDir.resolve(Paths.get(d.path))).forEach(p -> {
                 if (Files.notExists(p)) {
                     Logger.error(String.format("Managed directory does not exist: %s", p));
-                    System.exit(1);
+                    System.exit(0);
```

Comment:
This is an unrecoverable error, it really should exit with a non zero code.

Reply:
Yep because with an error 0 the terminal closes too quickly to see what happened. I wasted a bit of time thinking my build was no good when in fact I was missing the “mods” folder.

Done here: https://github.com/superzanti/ServerSync/pull/319/commits/944701eb51ca35bc56bf34336d550373f5ae5362

Reply:
could probably put a sleep or pause in here before exiting if that is useful.